app/mixins/subscribe.js

```diff
@@ -160,6 +165,17 @@ export default Ember.Mixin.create({
   // WebSocket connected
   subscribeConnected: function(tries,msec) {
     this.set('connected', true);
+    this.set('wasConnected', true);
+
+    if( this.get('warningShown') ) {
+      this.get('growl').close();
+      this.set('warningShown', false);
+    }
+
+    if ( this.get('disconnectedTimer') )
```

Comment:
Cancel does nothing if given null/an invalid timer, so the `if()` wrapper isn't needed